salt/pillar/data-pipeline.sls

```diff
@@ -11,6 +11,11 @@ data_pipeline:
             client_id: dummy-client-id
             client_secret: dummy-client-secret
 
+        sensitive_props:
+            key: ""
```

Comment:
only `key` is actually sensitive in here, the rest is like saying `TLS 1.1`; but they are all nicely grouped together and should be in pillars. Would just called them `cryptography_props` maybe?

Reply:
they're called `sensitive-props` in the nifi configuration file, I'll stick with that for now.

from the documentation I'm under the impression that the values are used together to perform the encryption, with the `key` property being the only *really* sensitive value.

Reply:
Good if it's a terminology coming from Nifi